libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/BaseGradleIT.kt

```diff
@@ -274,6 +277,20 @@ abstract class BaseGradleIT {
         }
     }
 
+    fun <T> Project.getModels(modelType: Class<T>): ModelContainer<T> {
+        if (!projectDir.exists()) {
+            setupWorkingDir()
+        }
+
+        val connection = GradleConnector.newConnector().forProjectDirectory(projectDir).connect()
+        val options = defaultBuildOptions()
+        val arguments = mutableListOf("-Pkotlin_version=${options.kotlinVersion}")
+        options.androidGradlePluginVersion?.let { arguments.add("-Pandroid_tools_version=$it") }
+        val model = connection.action(ModelFetcherBuildAction(modelType)).withArguments(arguments).run()
```

Comment:
It looks like `options.androidHome` should be passed here as well (using the `ANDROID_HOME` environment variable), otherwise the test project builds fail on a machine without system-wide `ANDROID_HOME`.

Reply:
Done. Thank you for catching this